### PR TITLE
feat: add iTerm2 color scheme

### PR DESCRIPTION
**Pull request by vaporwhere**
This PR includes the iTerm 2 color scheme, installation instructions, and a screenshot showcasing the theme in the same style as the Terminal one.

### CODE REVIEW

File: resources/iTerm2/readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Remedy - Terminal
```
Comment (robertrossmann) [inline resources/iTerm2/readme.md]: I think this should be `Remedy - iTerm 2`, right?

File: resources/iTerm2/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+> An iTerm 2 profile matching the colours used in VS Code's Remedy theme. 🎨
+
+## Installation
+
+- Download the _Remedy.itermcolors_ file from the [latest release assets][github-releases] on Github or from this directory
```
Comment (robertrossmann) [inline resources/iTerm2/readme.md]: Hey we need to add the profile to the assets. 😄 You can set that up in the _release.config.js_. It's pretty self-explanatory. 👍

File: resources/iTerm2/readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- Click on **iTerm2** menu title
+- Select **Preferences...** option
+- Select **Profiles**
+- Navigate to **Colors** tab
+- Click on **Color Presets**
+- Click on **Import**
+- Select the _Remedy.itermcolors_ file
+- Click on **Color Presets** and choose the Remedy color scheme
+
+After the installation you will likely want to fix/change the following settings:
```
Comment (robertrossmann) [inline resources/iTerm2/readme.md]: I am not sure this section applies to iTerm. With a Terminal profile you have to export your complete Terminal configuration, including your chosen font, cursor appearance and other non-colour-related options but with iTerm I see only colours defined. Perhaps we can drop this?

Comment (vaporwhere) [reply resources/iTerm2/readme.md]: That's right, iTerm only exports colors. I left that section so that folks know what to change to make it look like the screenshot, and also to be consistent with the Terminal readme. So I would reword this into:

> After the installation you might want to customize the following settings:
> - Font (I use _Operator Mono_ with ligatures hacked into it 🤷‍)
> - Cursor (I' using 'Box')
> - Any other, non-colour-related settings

If you think it doesn't add much/doesn't apply, I can remove it!

File: resources/iTerm2/readme.md (hunk 2 above)
Comment (vaporwhere) [reply resources/iTerm2/readme.md]: Sorry, completely missed this - never worked with semantic-release before! Fixed with cb50c20.

File: resources/iTerm2/readme.md (hunk 1 above)
Comment (vaporwhere) [reply resources/iTerm2/readme.md]: Fixed with 06f888a!

File: resources/iTerm2/readme.md (hunk 3 above)
Comment (robertrossmann) [reply resources/iTerm2/readme.md]: Sounds good! I would start the section with the following line, to emphasise that the next steps are only relevant if you want to make it look like in the screenshot.

> If you would like to make iTerm look like in the screenshot, you might want to customise the following settings:

(the rest as you wrote in the comment)

Comment (vaporwhere) [reply resources/iTerm2/readme.md]: Perfect! Fixed in e07d66a

**Comment by robertrossmann** [comment]
Thanks a lot for this contribution, @vaporwhere! ❤️
